### ISSUE TITLE
Bower dependencies and travis build are broken after split-shadycss  merge

### ISSUE THREAD

**Issue by tomalec** [webcomponents/shadycss#54]
https://travis-ci.org/webcomponents/shadycss/builds/202936639

Fails due to:
```log
bower ENORESTARGET  Tag/branch split-shadycss does not exist
Additional error details:
No tags found in https://github.com/webcomponents/shadydom.git
Available branches: disable-importNode, fix-es5, master, patching
The command "bower install" failed and exited with 1 during .
```

**Comment by tomalec** [comment]
@sorvell fix is pending at https://github.com/webcomponents/shadycss/pull/55

**Comment by azakus** [comment]
Fixed with 8754d5e5a821e56af9d83120602569d2a740bb87 and d6dd27eece94c0ee1005047b23c9c07786ef37c7
